install_config/aggregate_logging_sizing.adoc

```diff
@@ -0,0 +1,253 @@
+[[contributing-to-docs-templates-topic-template]]
+= Aggregate logging sizing guidelines 
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+The https://github.com/openshift/openshift-docs/blob/master/install_config/aggregate_logging.adoc#overview[ElasticSearch, Fluentd and Kibana] (EFK) stack aggregates logs from nodes and applications running inside your openshift installation.
+Once deployed it uses http://www.fluentd.org/architecture[Fluentd] to aggregate event logs from all nodes, projects and pods into https://www.elastic.co/products/elasticsearch[Elasticsearch]. It also provides a centralized https://www.elastic.co/guide/en/kibana/current/introduction.html[Kibana] web UI where users and administrators can create rich visualizations and dashboards with the aggregated data.
+
+Fluentd https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html[bulk uploads] logs to an index, in json format, and ElasticSearch then routes your search requests to the appropriate shards.
+
+== Installation
+The general procedure for installing an aggregate logging stack in {product-title} is described in https://docs.openshift.com/enterprise/3.2/install_config/aggregate_logging.html[Aggregating Container Logs].
```

Comment:
I'm not sure if just replacing 3.2 with {product-version} would work as intended.
@vikram-redhat ?